### ISSUE TITLE
Accordion animation broken without determinate header height

### ISSUE THREAD

**Issue by vxsl** [EQWorks/lumen-labs#58]
`Accordion`'s expand/collapse animation goes away as soon as you add custom `classes` to `Accordion.Panel` that don't include a determinate height for the `details` element (the portion of the component that expands and collapses). The default value for `classes` is `{{ details: 'h-10' }}`.

Here is a minimal reproducible example: 
```jsx
<Accordion>
  <Accordion.Panel
    // classes={{ details: 'h-10' }} is the defaultProp
    classes={{}} // remove this line and you should see the animation come back
  >
    {/* content lacking determinate height */}
  </Accordion.Panel>
</Accordion>
```

**Comment by vxsl** [comment]
@sallkall it appears that transitioning `height: auto` is a long-standing CSS limitation. Not sure if we want to go down this rabbit hole as [many of the workarounds seem hacky.](https://stackoverflow.com/questions/3508605/how-can-i-transition-height-0-to-height-auto-using-css)

**Comment by sallkall** [comment]
> @sallkall it appears that transitioning `height: auto` is a long-standing CSS limitation. Not sure if we want to go down this rabbit hole as [many of the workarounds seem hacky.](https://stackoverflow.com/questions/3508605/how-can-i-transition-height-0-to-height-auto-using-css)

right I remember this issue... @hyx131 what did you end up doing for accordions in QL?

**Comment by vxsl** [comment]
fix in #62
